packages/components/src/external-link/style.scss

```diff
@@ -0,0 +1,11 @@
+.components-external-link {
+	text-decoration: none;
+}
+
+.components-external-link__contents {
+	text-decoration: underline;
+}
+
+.components-external-link__icon {
+	margin-left: 0.5ch;
```

Comment:
Isn't that still unpredictable and varying? I wonder if we'd instead prefer a more fixed value that doesn't change based on the font.

Reply:
Not a strong feeling, but I like that using `ch` means the spacing is relative to the font. 

If we used a fixed space then the gap could end up feeling too large for condensed font variants, or too small for extended ones.

Reply:
Sounds good, as long as it's intentional 👍